website/assets/scss/_styles_project.scss

```diff
@@ -1 +1 @@
-@import 'td/code-dark';
\ No newline at end of file
+@import 'td/code-dark';
```

Comment:
This needed for [code highlighting with theme switch](https://www.docsy.dev/docs/adding-content/lookandfeel/#lightdark-color-themes).